src/Protocol/Models/DidChangeConfigurationParams.cs

```diff
@@ -10,6 +11,6 @@ public class DidChangeConfigurationParams : IRequest
         /// <summary>
         ///  The actual changed settings
         /// </summary>
-        public IDictionary<string, BooleanNumberString> Settings { get; set; } = new Dictionary<string, BooleanNumberString>();
+        public JToken Settings { get; set; }
```

Comment:
Woot! 😄

Reply:
I feel like it changed at somepoint, I just never noticed it.  Keeping a reference spec in sync is hard...

Reply:
I bet!  You are doing an amazing job here working on this, and I very much appreciate your hard work!